lib/cell/escaped.rb

```diff
@@ -4,24 +4,27 @@ def self.included(includer)
   end
 
   module Property
-    def property(name, *args)
-      super.tap do # super defines #title
-        mod = Module.new do
-          define_method(name) do |options={}|
-            value = super() # call the original #title.
-            return value unless value.is_a?(String)
-            return value if options[:escape] == false
-            escape!(value)
+    def property *names
+      super.tap do
+        prepend Module.new {
```

Comment:
Let me prepare the `Escaped` module a bit for you.

Some suggestions for PRs:

* Try to keep in line with the author's style (e.g. I don't like `def property *names` without brackets).
* Don't use shit features like `prepend` - that was only added to Ruby because Rails messed up. You can achieve the same with well-tested and working Ruby mechanisms.
* Add tests when you change things.

Thanks!